It seems that on ubuntu there is a problem with the python 3 version of the annotator

Thus, include only the first imports, and make sure to use --hidden-import="PIL._tkinter_finder"
when running pyinstaller. Doing so resolves the error with ImageTk.

If python 3 doesn't run the pyinstaller because of maximum recursive depth, that's a problem
with python 3 on ubuntu but should run fine on macos.

It is possible that a small tinkering with imports has changed the code slightly. If so,
it is a minor error relating to the python 3 uncommented imports
